src/LanguageServer.php

```diff
@@ -190,6 +191,7 @@ public function initialize(ClientCapabilities $capabilities, string $rootPath =
             $this->projectIndex = new ProjectIndex($sourceIndex, $dependenciesIndex, $this->composerJson);
             $stubsIndex = StubsIndex::read();
             $this->globalIndex = new GlobalIndex($stubsIndex, $this->projectIndex);
+            $initializationOptions = $initializationOptions ?? new Options;
```

Comment:
If `$initializedOptions` is not `null`, it will be `stdClass` because it's not typed. I also think you don't account for the nesting under `->php` here